src/targets/gpu/driver/precompile_op.cpp

```diff
@@ -0,0 +1,84 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2015-2022 Advanced Micro Devices, Inc. All rights reserved.
```

Comment:
```suggestion
 * Copyright (c) 2015-2023 Advanced Micro Devices, Inc. All rights reserved.
```
Until we batch-process all files to update the year to 2023, we can at least use 2023 for the new files.